library/src/main/java/bio/terra/landingzone/stairway/flight/ParametersResolverProvider.java

```diff
@@ -0,0 +1,48 @@
+package bio.terra.landingzone.stairway.flight;
+
+import bio.terra.landingzone.library.configuration.LandingZoneAzureRegionConfiguration;
+import bio.terra.landingzone.library.landingzones.definition.factories.ParametersResolver;
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ParametersResolverProvider {
+
+  private final String GLOBAL_KEY = "global";
```

Comment:
I think creation should die if the provided region is not provided instead of falling back to GLOBAL. There is no guarantee that the global parameters are in fact globally supported. If there is clamor for many other regions which overwhelms our ability to vet and add them to the config perhaps we'd do a GLOBAL region then, but for now I think we should be fairly conservative.  Thoughts?